routers/wishlist-router.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+var bodyParser = require('body-parser');
+const WishlistRouter = express.Router();
+const {Wishlist} = require('../models/wishlist-model')
+// to parse the data coming from body as JSON we need to use body-parser
+
+const cors = require('cors')
+WishlistRouter.use(cors())
+// Middleware to make the data as JSON
+WishlistRouter.use(bodyParser.json({limit: '5000kb'}));
```

Comment:
Similarly for this syed ...bodyParser.json() will just put the body into req.And we always want to put body into request for every single route .So i think u should also move it to index.js .

Reply:
my bad, surely was redundant, changed it